addons/im_livechat/static/src/embed/common/livechat_button.js

```diff
@@ -43,6 +43,18 @@ export class LivechatButton extends Component {
             },
         });
         useExternalListener(document.body, "scroll", this._onScroll, { capture: true });
+        useEffect(
+            () => {
+                const rootNodeClassList = this.ref.el?.getRootNode().host?.classList;
```

Comment:
Shouldn't this be put as dependency also?

I don't know how realistic it is for that to change (and it won't be observed either way), but it looks like we would want to clean or run the effect again if it does change.